packages/babel-plugin/__tests__/import-mapping/__snapshots__/import-mapping.js.snap

```diff
@@ -1,5 +1,94 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`import mapping emotion-css-keyframes 1`] = `
+"import { keyframes } from 'package-three'
+
+keyframes\`
+  from, 20%, 53%, 80%, to {
+    transform: translate3d(0,0,0);
+  }
+
+  40%, 43% {
+    transform: translate3d(0, -30px, 0);
+  }
+
+  70% {
+    transform: translate3d(0, -15px, 0);
+  }
+
+  90% {
+    transform: translate3d(0,-4px,0);
+  }
+\`
+
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+function _EMOTION_STRINGIFIED_CSS_ERROR__() { return \\"You have tried to stringify object returned from \`css\` function. It isn't supposed to be used directly (e.g. as value of the \`className\` prop), but rather handed to emotion so it can handle it (e.g. as value of \`css\` prop).\\"; }
+
+import { keyframes } from 'package-three';
+
+/*#__PURE__*/
+keyframes(process.env.NODE_ENV === \\"production\\" ? {
+  name: \\"w3sfyc\\",
+  styles: \\"from,20%,53%,80%,to{transform:translate3d(0,0,0);}40%,43%{transform:translate3d(0, -30px, 0);}70%{transform:translate3d(0, -15px, 0);}90%{transform:translate3d(0,-4px,0);}\\"
+} : {
```

Comment:
It's not that simple - you would have to produce slightly different output here. Notice that the simplified object for `keyframes` is different than this:
https://github.com/emotion-js/emotion/blob/f3c51e8aca287170e43110bee2e4527eacfcada4/packages/react/src/keyframes.js#L13-L22

Reply:
Got, I try to fix it. Thanks 😄